storage/shard.go

```diff
@@ -305,6 +326,25 @@ func (s *dbShard) Flush(ps persist.Persister) error {
 // a given element. Ignore for now due to lack of proper API.
 // TODO(xichen): Propagate ticking stats back up.
 func (s *dbShard) Tick(ctx context.Context) error {
+	var (
+		numberOfLoadedSegments   int
+		numberOfUnloadedSegments int
+	)
+	if err := s.forEachSegment(func(segment sealedFlushingSegment) error {
```

Comment:
You can probably get this information returned from `tryUnloadSegments` somehow. `forEachSegment` requires locking the shard, so would be great if we can reuse the `forEachSegment` loop in `tryUnloadSegments`.